Create CODE_OF_CONDUCT.md (#78)
OBF/obf-docs: code-of-conduct/CODE_OF_CONDUCT.md

**Board members** are expected to adhere to this Code of Conduct, and could be ejected from the board (as per article 3, paragraph 4 of the OBF bylaws) for breaching the OBF Code of Conduct even when they are not representing OBF.

**OBF organised meetings**, such as OBF Board meetings (both public and Board-only), community calls and similar events. This does not include sponsored events where OBF is not the main organiser. It also does not include BOSC when held jointly and under a separate Code of Conduct, which will be stated clearly.

**BOSC Organising Committee members and BOSC abstract reviewers**, even in cases when the OBF Code of Conduct doesn’t apply to the conference itself (depending on conference co-location it may or may not be adopted).

**OBF Event fellowship awardees**, who are supported by OBF to attend an event and previous awardees when they represent OBF in an event or a community space.

**Mentors, organisers, participants, project leaders, interns, and students**, when participating in CoFest, GSoC, GSoD, or similar OBF-umbrella activities for the duration of the event, internship, mentorship and similar occasions.

> **Communication and community engagement platforms**, such as project repositories on GitHub (issues, PRs or other contributions), mailing list, website (blogs and comment sections), Gitter channel and workshops or training events, communication platforms such as Gitter, Slack and Twitter.

Author: ```suggestion
**Communication and community engagement platforms**, such as project repositories on GitHub (issues, PRs or other contributions), mailing list, website (blogs and comment sections), workshops or training events, and communication platforms such as Gitter, Slack and Twitter.
```

Author: @brianrepko can you please double-check this edit?

### 2.3 OBF Code of Conduct Committee

The volunteer members designated and approved by the OBF Board to serve on the Code of Conduct Committee will be responsible for enforcing the Code of Conduct. Any voting members of the OBF community including the liaisons from the OBF Board can volunteer to sit in the Code of Conduct committee. They will be required to attend a training workshop on Code of Conduct report handling and enforcement as part of the onboarding process. We seek to build a committee including members outside of the OBF Board. Please contact the OBF board to express your interest to join this committee. The current members are listed in the OBF Code of Conduct document.

Reports to the Code of Conduct Committee should be made by emailing `<dedicated Code of Conduct email>`. Currently appointed members to report any incident of violation of Code of Conduct are Bastian Greshake Tzovaras, Malvika Sharan and Yo Yehudi. They are also available to respond to any question and concerns regarding the Code of Conduct document and procedure.

All reports will be reviewed by the Code of Conduct Committee and will be kept confidential.

### 2.4 Expected Behavior

> All participants in our events and communications are expected to show respect and courtesy to others. All interactions should be professional regardless of platform: either online or in-person. In order to foster a positive and professional learning environment we encourage the following kinds of behaviors in all OBF events and platforms:

Author: ```suggestion
All participants in our events and communications are expected to show respect and courtesy to others. All interactions should be professional regardless of platform: either online or in-person.
```

We are dedicated to providing a welcoming and supportive environment for all people, regardless of background or identity. By participating in this community, participants agree to abide by the OBF's Code of Conduct and accept the procedures by which any Code of Conduct incidents are resolved. Any form of behavior to exclude, intimidate, or cause discomfort is a violation of the Code of Conduct. In order to foster a positive and professional learning environment we encourage the following kinds of behaviors in all platforms and events:

* Use welcoming and inclusive language
* Be respectful of different viewpoints and experiences
* Gracefully accept constructive criticism
* Focus on what is best for the community
* Show courtesy and respect towards other community members

> Please contact the Code of Conduct Committee by emailing `dedicated Code of Conduct email`.

Reviewer: ```suggestion
Please contact the Code of Conduct Committee by emailing `coc@open-bio.org`.
```

The Code of Conduct Committee is established with the volunteer members designated and approved by the OBF Board. The committee will be responsible for reviewing and keeping confidentiality of all the reports when enforcing the Code of Conduct.

## 2. OBF Code of Conduct in detail

### 2.1 Code of Conduct statement

> In OBF, we do not tolerate behavior that is disrespectful to our community members or that excludes, intimidates, or causes discomfort to others. We do not tolerate discrimination or harassment based on characteristics that include but are not limited to, gender identity and expression, sexual orientation, disability, physical appearance, body size, citizenship, nationality, ethnic or social origin, pregnancy, familial status, veteran status, genetic information, religion or belief (or lack thereof), membership of a national minority, property, age, education, socio-economic status, technical choices, and experience level. 

Author: We took that directly from the reference, but I think it is better to remove that as I can't explain what it really means,
.

Everyone who participates in the OBF events or activities is required to conform to this Code of Conduct. It applies to all spaces managed by OBF as described in the ‘Scope of the OBF Code of Conduct’. Event hosts are expected to assist with the enforcement of the Code of Conduct and report all incidents to the designated Code of Conduct committee. By participating, participants indicate their acceptance of the procedures by which OBF resolves any Code of Conduct incidents, which may include storage and processing of their personal information. 

### 2.2 Scope of the OBF Code of Conduct (Code of Conduct)

> To make clear what is expected, everyone participating in OBF activities is required to conform to our Code of Conduct.

Author: Removed.

**Communication and community engagement platforms**, such as project repositories on GitHub (issues, PRs or other contributions), mailing list, website (blogs and comment sections), Gitter channel and other communication platforms, workshops or training events, online forums such as Gitter chat rooms and Twitter.

### 2.3 OBF Code of Conduct Committee

The volunteer members designated and approved by the OBF Board to serve on the Code of Conduct Committee will be responsible for enforcing the Code of Conduct. Any voting members of the OBF community including the liaisons from the OBF Board can volunteer to sit in the Code of Conduct committee. They will be required to attend a training workshop on Code of Conduct report handling and enforcement as part of the onboarding process. We seek to build a committee including members outside of the OBF Board. Please contact the OBF board to express your interest to join this committee. The current members are listed in the OBF Code of Conduct document.

> Reports to the Code of Conduct Committee should be made by emailing `<dedicated Code of Conduct email>`. Currently appointed members to report any incident of violation of Code of Conduct are Bastian Greshake Tzovaras, Malvika Sharan and Yo Yehudi. They are also available to respond to any question and concerns regarding the Code of Conduct document and procedure.

Reviewer: ```suggestion
Reports to the Code of Conduct Committee should be made by emailing `coc@open-bio.org`. Currently appointed members to report any incident of violation of Code of Conduct are Bastian Greshake Tzovaras, Malvika Sharan and Yo Yehudi. They are also available to respond to any question and concerns regarding the Code of Conduct document and procedure.
```

All reports will be reviewed by the Code of Conduct Committee and will be kept confidential.

### 2.4 Expected Behavior

All participants in our events and communications are expected to show respect and courtesy to others. All interactions should be professional regardless of platform: either online or in-person. In order to foster a positive and professional learning environment we encourage the following kinds of behaviors in all OBF events and platforms:

* Use welcoming and inclusive language
* Be respectful of different viewpoints and experiences
* Gracefully accept constructive criticism
* Focus on what is best for the community

> * Show courtesy and respect towards other community members

Reviewer: This seems redundant with the Overview section. Does this really need to be in both places?

Author: No. I can edit that.

We also subscribe to the four social rules of the [Recurse Center](https://www.recurse.com/manual#sub-sec-social-rules): 
* No feigning surprise
* No "well-actually"
* No backseat-driving
* No subtle "-isms" 

### 2.5 Unacceptable behavior

Examples of unacceptable behavior by participants at any OBF event/platform include:

- written or verbal comments which have the effect of excluding people based on membership of any specific group  
- causing someone to fear for their safety, such as through stalking, following, or intimidation  
- violent threats or language directed against another person
- the display of sexual or violent images  
- unwelcome attention  
- continuing to initiate interaction (including photography or recording) with someone after being asked to stop
- nonconsensual or unwelcome physical contact  
- sustained disruption of talks, events or communications
- insults or put-downs  
- sexist, racist, homophobic, transphobic, ableist, or exclusionary language/jokes
- excessive swearing
- incitement to violence, suicide, or self-harm  
- sharing private communication without consent  

### 2.6 Consequences of Unacceptable Behavior

> Participants who are asked to stop any inappropriate behavior are expected to comply immediately. This applies to any OBF events and platforms, either online or in-person. If a participant engages in behavior that violates this Code of Conduct, the organizers may warn the offender, ask them to leave the event or platform (without refund), or engage OBF Code of Conduct Committee to investigate the Code of Conduct violation and impose appropriate sanctions as described in the reporting and report-handling guidelines.

Reviewer: ```suggestion
Participants who are asked to stop any inappropriate behavior are expected to comply immediately. This applies to any OBF events and platforms, either online or in-person. If a participant engages in behavior that violates this Code of Conduct, the organizers may warn the offender, ask them to leave the event or platform (without refund), or engage the Code of Conduct Committee to investigate the Code of Conduct violation and impose appropriate sanctions as described in the reporting and report-handling guidelines.
```
